test/DebugInfo/omit-empty.ll

```diff
@@ -20,7 +20,7 @@ target triple = "spir64-unknown-unknown"
 !4 = !{i32 2, !"Debug Info Version", i32 3}
 !5 = distinct !DICompileUnit(language: DW_LANG_C99, file: !1, producer: "LLVM", isOptimized: false, runtimeVersion: 0, emissionKind: FullDebug, imports: !6)
 !6 = !{!7}
-!7 = !DIImportedEntity(tag: DW_TAG_imported_module, scope: !8, entity: !8, file: !1, line: 3)
+!7 = !DIImportedEntity(tag: DW_TAG_imported_module, scope: !8, entity: !11, file: !1, line: 3)
```

Comment:
The test was synthetic and incorrect. In order to restore llvm_unreachable from above fixed the test.

Previous version of the test was looking like having using directive applied to a function, but there is no such feature in any languages. The closest would be functors and function pointers, but they have different IR generated by clang.